lib/dotcom_web/components/live_components/trip_planner_form.ex

```diff
@@ -151,6 +134,30 @@ defmodule DotcomWeb.Components.LiveComponents.TripPlannerForm do
     """
   end
 
+  defp search_box(assigns) do
```

Comment:
I like this abstraction! could we name it `location_search_box` or something else a bit less general than `search_box`?

Reply:
Done and done!